File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c#/Guillermoqnk.cs
namespace Reto1
{
    internal class Guillermoqnk
    {
        static void Main(string[] args)
        {
            for (int i = 0; i <= 100; i++)
            {
                if ((i % 3 == 0) && (i % 5 == 0))
                {
                    Console.WriteLine("fizzbuzz");
                }
                else if (i % 3 == 0)
                {
                    Console.WriteLine("fizz");
                }
                else if (i % 5 == 0)
                {
                    Console.WriteLine("buzz");
                }
                else
                {
                    Console.WriteLine(i);
                }
            }
        }
    }
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c/Guillermoqnk.c
int main()
{
    for (int i = 1; i <= 100; ++i)
    {
        if(i%3 == 0 && i%5 == 0){
            printf("fizzbuzz ");
        }
        else if(i%3 == 0){
            printf("fizz ");
        }
        else if(i%5 == 0){
            printf("buzz ");
        }
        else{
            printf("%d ", i);
        }
    }
    
}